autogalaxy/operate/deflections.py

```diff
@@ -709,6 +726,29 @@ def einstein_mass_angular_from(
 
         return einstein_mass_angular_list[0]
 
+    def jacobian_stack(self, y, x, pixel_scales):
+        if not use_jax:
+            return

```

Comment:
Should this ever get called if jax is off?

Reply:
I don't expect it to, but figured just to be safe.

Reply:
Might be worth raising an exception instead?

Reply:
Something like: 

```python
if not use_jax:
    raise AssertionError("Function should not be called unless JAX is in use")

return ...